tick/inference/tests/linear_regression_test.py

```diff
@@ -55,7 +55,9 @@ def test_LinearRegression_fit(self):
                                        fit_intercept=fit_intercept,
                                        solver=solver,
                                        penalty=penalty,
-                                       tol=1e-10)
+                                       tol=1e-10,
+                                       blocks_start=[0],
+                                       blocks_length=[1])
```

Comment:
You cannot add `blocks_start=[0], blocks_length=[1]` just because it's convenient :)